admin/view/wp-slimstat-db.php

```
@@ -1516,4 +1516,279 @@ public static function get_your_blog()
+                $table_stats,
+                $prepared_where,
+                $date_where
+            );
+        }
+
+        // Check transient cache (1-hour TTL, keyed by goal + date range)
+        $cache_key = 'slimstat_goal_' . md5($sql);
+        $result    = get_transient($cache_key);
+
+        if (false === $result) {
+            $result = wp_slimstat::$wpdb->get_row($sql, ARRAY_A);
```

PHPStan (2.1.46)

[error] Constant ARRAY_A not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)